misc/msvs/vcxproj.template

```diff
@@ -37,4 +37,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
This needs to be tested on Windows with the new vsproj generation to make sure it still works fine with the trailing newlines.
Otherwise we'll need to exclude these files from the check. CC @shana